docs/setup-hybrid-OVNKubernetes-cluster.md

```diff
@@ -1,7 +1,7 @@
 # Setup hybrid OVNKubernetes cluster
 
 This guide assumes the user has installed current versions of the OpenShift installer (`openshift-install`) and client (`oc`) binaries.
-Please refer to the [official OpenShift Container Platform documentation](https://docs.openshift.com/container-platform/4.5/welcome/index.html) for details.
+Please refer to the [official OpenShift Container Platform documentation](https://docs.openshift.com/container-platform/latest/welcome/index.html) for details.
```

Comment:
you're setting some of these to latest and some to 4.11.
personally, i'd prefer all to `latest` as that would save us time in the future, and whoever clicks the link can just select their version on the doc page

Reply:
Here's my take: the ones in the CSV are versioned since it's tied to a specific WMCO release, the ones in our internal docs only are `latest`